tests/cephfs/nfs/move_data_bw_nfs_and_cephfs_mounts.py

```diff
@@ -0,0 +1,194 @@
+        nfs_mounting_dir = f"/mnt/cephfs_nfs{mounting_dir}_1/"
+        client1.exec_command(sudo=True, cmd=f"mkdir -p {nfs_mounting_dir}")
+        command = f"mount -t nfs -o port=2049 {nfs_server}:{nfs_export_name} {nfs_mounting_dir}"
+        output, err = client1.exec_command(sudo=True, cmd=command, check_ec=False)
+        if build.startswith("5"):
+            kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+            mon_node_ips = fs_util.get_mon_node_ips()
+            fs_util.kernel_mount(
+                [clients[0]],
+                kernel_mounting_dir_1,
+                ",".join(mon_node_ips),
+                extra_params=f",fs={default_fs}",
+            )
+
+            fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+            fs_util.fuse_mount(
+                [clients[0]],
+                fuse_mounting_dir_1,
+                extra_params=f" --client_fs {default_fs}",
+            )
+        else:
+            kernel_mounting_dir_1 = f"/mnt/cephfs_kernel{mounting_dir}_1/"
+            mon_node_ips = fs_util.get_mon_node_ips()
+            fs_util.kernel_mount(
+                [clients[0]],
+                kernel_mounting_dir_1,
+                ",".join(mon_node_ips),
+            )
+
+            fuse_mounting_dir_1 = f"/mnt/cephfs_fuse{mounting_dir}_1/"
+            fs_util.fuse_mount(
+                [clients[0]],
+                fuse_mounting_dir_1,
+            )
+
+        run_ios(
+            clients[0],
+            kernel_mounting_dir_1,
+            file_name="dd_file_kernel",
+            bs="100M",
+            count=20,
+        )
+        run_ios(
+            clients[0],
+            fuse_mounting_dir_1,
+            file_name="dd_file_fuse",
+            bs="100M",
+            count=20,
+        )
+        run_ios(
+            clients[0], nfs_mounting_dir, file_name="dd_file_nfs", bs="100M", count=20
+        )
+
+        log.info("Migrate data b/w NFS mounts and CephFS mounts(kernel and fuse)")
+        kernelpath = f"{kernel_mounting_dir_1}{clients[0].node.hostname}dd_file_kernel"
+        fusepath = f"{fuse_mounting_dir_1}{clients[0].node.hostname}dd_file_fuse"
+        nfspath = f"{nfs_mounting_dir}{clients[0].node.hostname}dd_file_nfs"
+        mv_bw_mounts = [
+            f"cp {kernelpath} {nfs_mounting_dir}_dd_file_kernel",
+            f"cp {fusepath} {nfs_mounting_dir}_dd_file_fuse",
+            f"cp {nfspath} {kernel_mounting_dir_1}_dd_file_nfs1",
+            f"cp {nfspath} {fuse_mounting_dir_1}_dd_file_nfs2",
+        ]
+        for cmd in mv_bw_mounts:
+            clients[0].exec_command(sudo=True, cmd=cmd)
+
+        log.info("Confirm if data b/w mounts are migrated")
+        verify_data_movement = [
+            f" ls -l {nfs_mounting_dir}_dd_file_kernel",
+            f" ls -l {nfs_mounting_dir}_dd_file_fuse",
+            f" ls -l {kernel_mounting_dir_1}_dd_file_nfs1",
+            f" ls -l {fuse_mounting_dir_1}_dd_file_nfs2",
+        ]
```

Comment:
Hi Hemanth,
Can we use diff instead of ls .
diff -qr {kernelpath} {nfs_mounting_dir}_dd_file_kernel.
It will compare data as well.

Reply:
Updated